benchmark/benchmark.js

```diff
@@ -1,13 +1,10 @@
-#!/usr/bin/env node
```

Comment:
I'd keep the shebang since the script is a CLI script

Reply:
`eslint-plugin-node`'s recommended config only allows a shebang when the script is actually a part of the `package.json`'s `bin`s. There isn't too much of a point for exposing this externally, so removing it seems reasonable. We always call this with `node` anyway, so nothing should change.

Reply:
Yeah, exactly why the shebang should stay :)

IMO this is wrong, and the file should be treated as a CLI script since that's what it is.

Reply:
See also https://github.com/mysticatea/eslint-plugin-node/issues/96

and https://github.com/xojs/xo/blob/master/config/plugins.js#L309-L310

Personally, I'd disable the rule and keep the shebang.

Reply:
Thanks a lot for the references. This was super helpful, I'm convinced 😄